### ISSUE TITLE
Syura Graph Feature Doesn't Work for Contracts with Named Mappings (0.8.18+)

### ISSUE THREAD

**Issue by RealJohnnyTime** [Consensys/vscode-solidity-auditor#137]
When enabling the graph feature in a solidity contract of version 0.8.18 or higher, and if there exists a named mapping like `mapping(address token => address owner) private data;` the graph feature fails to function without providing any error messages. It simply does not generate the graph.

Has anyone else encountered this issue?

**Comment by IAm0xAxe** [comment]
Yeah,I've met too.

**Comment by spaceh3ad** [comment]
Yeah, I have same error. Though I think it's Surya related problem since with cmd it also fails to produce graphs.
It produces but empty images.

**Comment by GNSPS** [comment]
Possibly related to https://github.com/Consensys/surya/issues/190, which was just fixed in Surya v0.4.9.
